util-stats/src/main/scala/com/twitter/finagle/stats/InMemoryStatsReceiver.scala

```diff
@@ -114,7 +116,16 @@ class InMemoryStatsReceiver extends StatsReceiver with WithHistogramDetails {
 
       def add(value: Float): Unit = stats.synchronized {
         val oldValue = apply()
-        stats(schema.metricBuilder.name) = oldValue :+ value
+
+        // Limit the maximum number of stats as 10000000
```

Comment:
where does `10000000` come from?

Reply:
Oh, it came from my experiment code. Just removed thet comment.